.github/workflows/trilogy.yml

```diff
@@ -43,8 +43,8 @@ jobs:
           MYSQL_ROOT_PASSWORD: root
       - name: Create user, grant privledges,and flush # workaround to resolve https://github.com/trilogy-libraries/activerecord-trilogy-adapter/issues/64
         run: |
-          mysql -e "CREATE USER 'test'@'localhost' IDENTIFIED WITH mysql_native_password BY 'test';" -u${{ env.MYSQL_USER }} -p${{ env.MYSQL_PASSWORD }}
-          mysql -e "GRANT ALL PRIVILEGES ON *.* TO 'test'@'localhost' WITH GRANT OPTION;" -u${{ env.MYSQL_USER }} -p${{ env.MYSQL_PASSWORD }}
-          mysql -e "FLUSH PRIVILEGES;" -u${{ env.MYSQL_USER }} -p${{ env.MYSQL_PASSWORD }}
+          mysql -h 127.0.0.1 -e "CREATE USER 'test'@'localhost' IDENTIFIED WITH mysql_native_password BY 'test';" -u${{ env.MYSQL_USER }} -p${{ env.MYSQL_PASSWORD }}
+          mysql -h 127.0.0.1 -e "GRANT ALL PRIVILEGES ON *.* TO 'test'@'localhost' WITH GRANT OPTION;" -u${{ env.MYSQL_USER }} -p${{ env.MYSQL_PASSWORD }}
+          mysql -h 127.0.0.1 -e "FLUSH PRIVILEGES;" -u${{ env.MYSQL_USER }} -p${{ env.MYSQL_PASSWORD }}
```

Comment:
@cfis do you know if there's a way for me to run the GitHub actions on my own fork? I'm making educated guesses at the Github action issues around these mysql commands.

Reply:
I think via the .github/workflows/trilogy.yml file. But since that is part of the PR maybe it doesn't work (might have to be merged first?)

Reply:
@cfis I can spin up a new PR and see if merging in other components first resolve this issue.

Reply:
@cfis any objections in me breaking this into two PR's?

Reply:
Works for me. Sorry this taking so long to merge. I can try and focus on it next week.

Reply:
First PR is here: https://github.com/composite-primary-keys/composite_primary_keys/pull/610

Reply:
I see two more PRs after that one. The first with the trilogy guts and the second being the GitHub action. How do you feel about that?